.changes/snapshot-migraion-v3age-zeroize.md

```diff
@@ -0,0 +1,13 @@
+---
+
+"iota-stronghold": minor
+"stronghold-engine" : minor
+"runtime" : minor
```

Comment:
I think the crate is called `stronghold-runtime`

Reply:
Done